wiki/Contests/en.md

```diff
@@ -37,11 +37,11 @@ The **Monthly Beatmapping Contest**, in its initial concept, was a mapping conte
 
 After a long hiatus, the Monthly Beatmapping Contest was rebooted in January 2020 with a brand new schedule, ruleset, voting procedure, and reward scheme. Many aspects of the competition were dramatically altered to make the contests more engaging and attractive, in which some of them were as follows:
 
-- Instead of a single song, the contest now offers a selection of songs from different [featured artists](/wiki/Featured_Artists) each month, in which all participating mappers are free to map and submit their entries for.
+- Instead of a single song, the contest now offers a selection of songs from different [Featured Artists](/wiki/Featured_Artists) each month, in which all participating mappers are free to map and submit their entries for.
```

Comment:
```suggestion
- Instead of a single song, the contest now offers a selection of songs from different [featured artists](/wiki/Featured_Artists) each month, in which all participating mappers are free to map and submit their entries for.
```